### DISCUSSION TITLE
GitHub for Android missing Restore link for archived issues in projects

### DISCUSSION THREAD

**Discussion by archon810** [community/community#74454] [Mobile]
### Select Topic Area

Bug

### Body

I just reopened a completed issue which was archived in our project, and GitHub for Android just showed me that it was archived without the ability to unarchive it. I had to go to the web to restore it.

Please add the missing Restore button to the Android app.

**Comment by eliperkins** [comment]
Hey @archon810 , thanks for this feedback! I can understand that bouncing out to the browser to click one button might be frustrating. I've created an issue in our internal tracking repo for this feature request. I'll let you know when this has been prioritized!
